api/res/openapi.yml

```diff
@@ -239,6 +279,19 @@ components:
       required: true
       schema:
         type: string
+    preferAsync:
+      in: header
+      name: Prefer
+      description: >-
+        Whether the server should respond without waiting for completion
+        (see [RFC-7240](https://tools.ietf.org/html/rfc7240) for details)
+      required: false
+      schema:
+        enum:
+          - respond-async
+          - respond-async, wait=1
+          - respond-async, wait=5
+          - respond-async, wait=10
```

Comment:
Shouldn't we move this examples to the example block? `wait` isn't limited to these examples.

Reply:
Yeah, it's kinda hard to model the header in a meaningful way.
Another method I could think of would be:
```
      example:
        respond-async,wait=10
      schema:
        type: array
        uniqueItems: true
        items:
          oneOf:
            - type: string
              description: Respond before the operation is finished
              pattern: ^respond-async$
              example: respond-async
            - type: string
              description: Wait the specified amount of seconds for the operation to finish
              pattern: ^wait=(\d+)$
              example: wait=10
```

Swagger UI can even validate it an provide a semi-useful UI (list of strings):
<img width="490" alt="image" src="https://user-images.githubusercontent.com/748636/114401021-d413b280-9ba2-11eb-90df-03d6c46d28f0.png">

Reply:
Looks good to me.